app/models/manageiq/providers/google/inventory/collector.rb

```diff
@@ -6,40 +6,56 @@ def initialize(_manager, _target)
     super
   end
 
-  def connection
-    @connection ||= manager.connect
+  def compute
+    @compute ||= manager.connect(:service => "compute")
+  end
+
+  def sql
+    @sql ||= manager.connect(:service => "sql")
   end
 
   attr_reader :project_key_pairs
 
   def availability_zones
-    connection.zones.all
+    compute.zones.all
+  end
+
+  def cloud_databases
+    @cloud_databases ||= sql.instances.all
+  rescue Google::Apis::ClientError # Catch an API exception if the sqladmin API isn't enabled
```

Comment:
I wonder if we can drop a warning in the log?  Or will it be too verbose?

Reply:
Yeah I considered that but given this isn't required a warning isn't necessarily accurate.  If you don't use gce cloud databases seeing a warning every refresh would be annoying.

And if you do use gce cloud databases then you won't hit this issue because the service will be enabled.

We can doc what you have to do to enable the service.